src/default-xpring-client.ts

```diff
@@ -1,9 +1,7 @@
-import { XpringClientDecorator } from "./xpring-client-decorator";
-import TransactionStatus from "./transaction-status";
-import { Wallet } from "xpring-common-js";
-import { TransactionStatus as RawTransactionStatus } from "../generated/legacy/transaction_status_pb";
-
-/* global BigInt */
+import { Wallet } from 'xpring-common-js'
+import { XpringClientDecorator } from './xpring-client-decorator'
+import TransactionStatus from './transaction-status'
+import { TransactionStatus as RawTransactionStatus } from '../generated/legacy/transaction_status_pb'
 
 // TODO(keefertaylor): Re-enable this rule when this class is fully implemented.
```

Comment:
I think we should just disable these under the TODO to re-enable these rules when completed
```suggestion
 // TODO(keefertaylor): Re-enable these rules when this class is fully implemented.
/* eslint-disable @typescript-eslint/require-await */
/* eslint-disable @typescript-eslint/no-empty-function */
/* eslint-disable @typescript-eslint/no-useless-constructor */
/* eslint-disable @typescript-eslint/no-unused-vars */
/* eslint-disable class-methods-use-this */
/* eslint-disable max-classes-per-file */
```